borg/archive.py

```diff
@@ -503,7 +501,10 @@ def process_file(self, path, st, cache):
             else:
                 self.hard_links[st.st_ino, st.st_dev] = safe_path
         path_hash = self.key.id_hash(os.path.join(self.cwd, path).encode('utf-8', 'surrogateescape'))
+        first_run = not cache.files
         ids = cache.file_known_and_unchanged(path_hash, st)
+        if first_run:
+            logger.info('processing files')
```

Comment:
the problem is that when we start processing files, we not only process files, but we also start checking the cache on the first process_files() call. so if we emit the "processing" files message in do_create, it actually looks like:

```
processing files
doing some long stuff with the cache
processing files actually starts here
```

which is pretty confusing...